boards/samr21-xpro/Kconfig

```diff
@@ -24,3 +24,4 @@ config BOARD_SAMR21_XPRO
     select HAS_RIOTBOOT
 
     select HAVE_SAUL_GPIO
+    select HAVE_EDBG_EUI
```

Comment:
Based on the test this seems to be the only supported platform, but I was not sure...

Reply:
I think it's only on samr21-xpro where the edbg also provides an EUI.

Reply:
Should I make it dependent on the board?

Reply:
No, there might be some other boards that have this feature, I just don't know about them.